packages/zambdas/src/ehr/create-update-medication-order/validateRequestParameters.ts

```diff
@@ -39,7 +39,9 @@ export function validateRequestParameters(
     if (missedFields.length > 0) throw new Error(`Missing fields in orderData: ${missedFields.join(', ')}`);
   }
 
-  validateInteractions(orderData.interactions);
+  if (orderData.interactions) {
```

Comment:
```suggestion
  if (orderData?.interactions) {
```
it will not work without this mark